populate_dev_data_handler.go

```diff
@@ -23,42 +23,49 @@ import (
     "appengine/datastore"
 )
 
+// Create TestRun entities for local development and testing.
+// These point to JSON files stored in /static/.
+// Key names for each entity are specified to make this route idempotent.
 func populateDevData(w http.ResponseWriter, r *http.Request) {
     ctx := appengine.NewContext(r)
 
-    // Add Chrome TestRun
-    testRun := &TestRun{
-        Revision: "e2d2cde03d",
-        BrowserName: "chrome",
-        BrowserVersion: "60.0",
-        OSName: "debian",
-        OSVersion: "8",
-        ResultsURL: "/static/chrome-60.0-debian-8.json",
-        CreatedAt: time.Now(),
+    devData := map[string]*TestRun{
```

Comment:
Noob Go question:

How should I read `map[string]*TestRun`? I guess it's a map of strings to TestRun pointers? It seems somehow strange that *TestRun is after the brackets, since in other languages like C++ this would be more like `map<string, *TestRun>`.